src/v/net/probes.cc

```diff
@@ -110,6 +111,28 @@ void server_probe::setup_metrics(
       });
 }
 
+void server_probe::setup_public_metrics(
+  ss::metrics::metric_groups& mgs, const char* proto) {
```

Comment:
`std::string_view` should be preferred over `const char*`.

Reply:
Done.